src/styles/variables.less

```diff
@@ -1,17 +1,79 @@
-@gutter: 20px;
-@drop-shadow: drop-shadow(0 4px 10px rgba(20,22,44,0.08));
+@font-face {
+  font-family: 'Circular Air Pro';
+  src: url('fonts/CircularAirPro-Light.eot');
+  src: url('fonts/CircularAirPro-Light.eot?#iefix') format('embedded-opentype'),
+  url('fonts/CircularAirPro-Light.woff') format('woff'),
+  url('fonts/CircularAirPro-Light.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Circular Air Pro';
+  src: url('fonts/CircularAirPro-Medium.woff2') format('woff2'),
+  url('fonts/CircularAirPro-Medium.woff') format('woff');
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Circular Air Pro';
+  src: url('fonts/CircularAirPro-Bold.eot');
+  src: url('fonts/CircularAirPro-Bold.eot?#iefix') format('embedded-opentype'),
+  url('fonts/CircularAirPro-Bold.woff') format('woff'),
+  url('fonts/CircularAirPro-Bold.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
```

Comment:
Is this a free font? Dunno if we should be distributing a paid font in a public repo like this.

Reply:
Good call